src/components/material/MaterialMainfestationItem.tsx

```diff
@@ -0,0 +1,111 @@
+    pid,
+    titles,
+    creators,
+    hostPublication,
+    languages,
+    identifiers,
+    contributors,
+    edition,
+    audience,
+    physicalDescriptions
+  }
+}) => {
+  const t = useText();
+  const [isOpen, setIsOpen] = useState(false);
+  const faustId = convertPostIdToFaustId(pid as Pid);
+
+  // TODO How to handle if there is no data?
+  const listDescriptionData = {
+    Type: { value: String(materialTypes?.[0]?.specific), type: "standard" },
+    Sprog: { value: String(languages?.main?.[0].display), type: "standard" },
+    Bidragsydere: { value: String(contributors?.[0].display), type: "link" },
+    Originaltitel: { value: String(titles?.original), type: "standard" },
+    ISBN: { value: String(identifiers?.[0].value), type: "standard" },
+    Udgave: { value: String(edition?.summary), type: "standard" },
+    Omfang: {
+      value: String(physicalDescriptions[0].numberOfPages),
+      type: "standard"
+    }, // No Dummy data
+    Forlag: { value: String(hostPublication?.publisher), type: "standard" },
+    Målgruppe: { value: String(audience?.generalAudience), type: "standard" }
+  };
+
+  return (
+    <div className="material-manifestation-item">
+      <div className="material-manifestation-item__availability">
+        {faustId && (
+          <AvailabilityLabel
+            manifestText={materialTypes[0]?.specific}
+            link="/"
+            faustIds={[faustId]}
+          />
+        )}
+      </div>
+      <div className="material-manifestation-item__cover">
+        <Cover pid={pid as Pid} size="small" animate={false} />
+      </div>
+      <div className="material-manifestation-item__text">
+        <h2 className="material-manifestation-item__text__title text-header-h4">
+          {titles?.main[0]}
+        </h2>
+        <p className="text-small-caption">
+          {t("materialHeaderAuthorByText")} {creators[0]?.display} (
+          {hostPublication?.year?.year})
+        </p>
+
+        <div
+          className={`material-manifestation-item__text__details ${
+            isOpen ? "expanded" : ""
+          }`}
+          onClick={() => {
+            setIsOpen(!isOpen);
+          }}
+          // TODO Check if this is the correct way to handle accessibility
+          onKeyPress={() => {
+            setIsOpen(!isOpen);
+          }}
+          role="button"
+          tabIndex={0}
+        >
+          <p className="link-tag text-small-caption">Detaljer om materialet</p>
+          <img src={ExpandIcon} alt="ExpandMore-icon" />
+        </div>
+        {isOpen && (
+          <ListDescription
+            className="mt-24"
+            data={listDescriptionData as ListData}
+          />
+        )}
+      </div>
+      <div className="material-manifestation-item__reserve">
+        <ButtonSmallFilled label="RESERVER" disabled={false} />
+        <span className="link-tag text-small-caption material-manifestation-item__reserve__find">
+          {t("findOnBookshelfText")}
+        </span>
```

Comment:
Consider adding a TODO here to show that we should make this button workable in the future. Alternatively just remove it.